pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -46,7 +46,10 @@ const (
 	MaxResourcePercentage = 100
 )
 
-func LowNodeUtilization(ctx context.Context, client clientset.Interface, strategy api.DeschedulerStrategy, nodes []*v1.Node, evictLocalStoragePods bool, podEvictor *evictions.PodEvictor) {
+func LowNodeUtilization(ctx context.Context, client clientset.Interface, strategy api.DeschedulerStrategy, nodes []*v1.Node, opts Options, podEvictor *evictions.PodEvictor) {
+	if !strategy.Enabled {
```

Comment:
This was removed by https://github.com/kubernetes-sigs/descheduler/pull/302

Reply:
Thanks, fixed.